packages/flutter/lib/src/widgets/binding.dart

```diff
@@ -859,6 +859,7 @@ mixin WidgetsBinding on BindingBase, ServicesBinding, SchedulerBinding, GestureB
       firstFrameCallback = (List<FrameTiming> timings) {
         assert(sendFramesToEngine);
         if (!kReleaseMode) {
+          developer.UserTag.defaultTag.makeCurrent();
```

Comment:
It might be worth mentioning what state you're clearing here so we can more easily find it later.

Reply:
done